packages/node/package.json

```diff
@@ -50,7 +52,7 @@
     "@types/cookie": "0.3.3",
     "@types/etag": "1.8.0",
     "@types/jest": "27.4.1",
-    "@types/node-fetch": "^2.6.1",
+    "@types/node-fetch": "^2.6.2",
```

Comment:
These types dont match the runtime version

Reply:
The latest version of `node-fetch` includes the type definitions. We no longer need this dependency I've removed it.